### ISSUE TITLE
homepage

### ISSUE THREAD

**Issue by marchidaniele** [NagiosEnterprises/nagioscore#1001]
Hi,

after upgrade 4.5.7 homepage is change. Where's the old page with pid and other info? Why background left panel is black?

![image](https://github.com/user-attachments/assets/f79c6094-b546-4d90-b21c-b962a306a094)

Is it intentional? Because there is no mention in the changelog.
Do you have any idea how to restore the situation?

**Comment by tsadpbb** [comment]
The changelog line is `* Update Exfoliation theme (Dylan Anderson)`

I'll see what I can do about the pid and adding a light mode (no promises) but the other links and their removal were intentional.

**Comment by ericloyd** [comment]
That is correct that the changelog indicates that the change was made, but there was no indication prior to the update that this change was going to be made in the first place.  Mucking about with Nagios Core interface is a dangerous edge to traverse.  Lots of people use Nagios Core because they have used it for years with a basically unchanging interface and they know exactly where everything is.  Making a change to the interface and making it the default is not a good look.  It should be an option that can be chosen, and it should be easy to go back to the previous interface if desired.

**Comment by marchidaniele** [comment]
> That is correct that the changelog indicates that the change was made, but there was no indication prior to the update that this change was going to be made in the first place. Mucking about with Nagios Core interface is a dangerous edge to traverse. Lots of people use Nagios Core because they have used it for years with a basically unchanging interface and they know exactly where everything is. Making a change to the interface and making it the default is not a good look. It should be an option that can be chosen, and it should be easy to go back to the previous interface if desired.

Totally agree. Please, consider our requests. @tsadpbb

**Comment by GregOriol** [comment]
I'd like to be able to go back to the normal color scheme for Nagios too

**Comment by tsadpbb** [comment]
#1013 basically does what @GregOriol posted but allows you to set dark vs light in configuration
